server/accounts.go

```diff
@@ -2889,6 +2885,21 @@ func (ur *URLAccResolver) Fetch(name string) (string, error) {
 	return string(body), nil
 }
 
+// Store will store or update the jwt claim of account `name`
+// to the base url, appending the account name onto the end.
+func (ur *URLAccResolver) Store(name, jwt string) error {
+	url := ur.url + name
+	resp, err := http.Post(url, "application/jwt", strings.NewReader(jwt))
+	if err != nil {
+		return err
+	} else if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("could not post to <%q>: %v", url, resp.Status)
```

Comment:
The directive `%q` is already surrounding the text with double quotes, so no need for `<>`.

Reply:
The `Fetch` method has the same format. I wanted to have same for `Store` method.

https://github.com/masudur-rahman/nats-server/blob/6685cd2ece64172caa206c5c83f8efd7245e2918/server/accounts.go#L2877-L2882

Reply:
Fair enough!